pubspec.yaml

```diff
@@ -1,39 +1,24 @@
 name: json_schema
 version: 1.0.7
```

Comment:
Do we want our "initial Workiva release" version? Edit: looks like the Changelog has it at 1.0.8

Reply:
Hoping MARV will bump this